### ISSUE TITLE
Route view map extends off page

### ISSUE THREAD

**Issue by bschoenfeld** [Code4HR/hrt-bus-finder#100]
Looks like height is set to 10 or 20 pixels too much on chrome on desktop. Looks ok in iOS Safari. 50/50 in chrome on iOS. Strange...
##

**Comment by DRD161** [comment]
I'd like to take a crack at this issue if that's ok :)

**Comment by qwo** [comment]
this is over a 4 year old issue so its worth confirming this is still a bug.

**Comment by DRD161** [comment]
Yeah it doen't appear as though the map extends off of the page. This issue can most likely be closed.
